src/server/apollo/resolvers/Query.ts

```diff
@@ -2,13 +2,31 @@ import { assertConferenceDay } from "../../../const";
 import { filterTalks } from "../../pretalx";
 import { talkTypeToSubmissionType } from "../utils";
 import { QueryResolvers } from "./__types__";
+import { addMinutes } from "../../../utils/addMinutes";
 
 export const Query: QueryResolvers = {
   talks: async (_root, args, { dataSources }) => {
     const talks = await dataSources.pretalx.getAllTalks();
 
+    let newTalks = [];
+
+    // Copy all the talks
+    for (var i = 0; i < talks.length; i++) {
```

Comment:
I'd prefer
```ts
const newTalks = talks.map((talk) => ({ ...talk }));
```

Nowadays, imperative array manipulation (e.g., using `.push`) has fallen out of style.